cmpserver/plugin/plugin.go

```diff
@@ -143,20 +143,45 @@ func environ(envVars []*apiclient.EnvEntry) []string {
 	return environ
 }
 
+// getTempDirMustCleanup creates a temporary directory and returns a cleanup function. The cleanup function panics if
+// cleanup fails. Use this function when failing to clean up the temporary directory is a security risk.
+func getTempDirMustCleanup(baseDir string) (workDir string, cleanup func(), err error) {
+	workDir, err = files.CreateTempDir(baseDir)
+	if err != nil {
+		return "", nil, fmt.Errorf("error creating temp dir: %s", err)
```

Comment:
let's use `%w` throughout this file too?

Reply:
I'll change it just on the lines that I touched. I'm a bit worried about review fatigue.  :-)

Reply:
Nevermind, it wasn't as many lines as I thought.